src/superphot_plus/fit_numpyro.py

```diff
@@ -512,17 +599,26 @@ def jax_guide(
     return np.array(post_reformatted_for_save).T
 
 
-def run_mcmc_batch(fns, t0_lim=None, plot=False):
-    """
-    Run dynesty importance nested sampling on datafile. Returns
-    set of equally weighted posteriors (sets of fit parameters).
+def run_mcmc_batch(filenames, t0_lim=None, plot=False):
```

Comment:
Docstring should say "numpyro's NUTS sampler" instead of "dynesty". This function specifically runs NUTS for a batch of light curves at the same time, apologies for confusion!

Reply:
So to double check - it should read as the following?
```
def run_mcmc_batch(filenames, t0_lim=None, plot=False):
    """Runs numpyro's NUTS sampler on data file to get a set of equally
    weighted posteriors (sets of fit parameters).
```

Reply:
Yes!